pkg/tbtc/heartbeat_failure.go

```diff
@@ -0,0 +1,41 @@
+package tbtc
+
+import (
+	"sync"
+)
+
+// heartbeatFailureCounter holds counters keeping track of consecutive
+// heartbeat failures. Each wallet has a separate counter. The key used in
+// the map is the uncompressed public key (with 04 prefix) of the wallet.
+type heartbeatFailureCounter struct {
```

Comment:
Let's move it to the `heartbeat.go` file. No need to create a separate file for one structure.

Reply:
Done in acd49ee1ec5326a0134c1554ea6c3f16be04dd9c.